src/frontend/src/handler/create_source.rs

```diff
@@ -899,12 +907,6 @@ pub(crate) async fn bind_source_pk(
         }
 
         (Format::Debezium, Encode::Json) => {
-            if !additional_column_names.is_empty() {
-                return Err(RwError::from(ProtocolError(format!(
-                    "FORMAT DEBEZIUM forbids additional columns, but got {:?}",
-                    additional_column_names
-                ))));
-            }
```

Comment:
There are other branches not covered: "(debezium, avro)", "(maxwell, json)", "(canal, json)", etc.